src/main/java/com/minecolonies/core/colony/ColonyManager.java

```diff
@@ -78,6 +78,11 @@ public final class ColonyManager implements IColonyManager
      */
     private boolean capLoaded = false;
 
+    /**
+     * Client side sound manager.
+     */
+    private SoundManager clientSoundManager;
```

Comment:
dont think a sound manager really belongs under colonymanager, since its a client side singleton why not just make it fully static? Could still do SoundManager.tick() from the hook here